assets/templates/analytics.html.tmpl

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+{{ template "header.html.tmpl" .}}
+</head>
+<body>
+{{ template "navbar.html.tmpl" .}}
+<div class="container">
+  <div class="row">
+    <div class="col-md-12">
+     <h2>Analytics</h2>
+    </div>
+    <p style="display:inline-block;" >Start Date: <input type = "text"  id="datepicker-start"></p>
+    <p style="display:inline-block;">End Date: <input type = "text"  id="datepicker-end"></p>
+    <button class="btn btn-default" id="get_parties_button">Get Historical</button>
+</div>
+</div>
+
+<div class="container">
+  <div class="row">
+    <div id="historical_parties" class="col-md-12">
+    <h2>Historical Parties</h2>
+    </div>
+</body>
+<script>
+$(function() {
+   $( "#datepicker-start" ).datepicker();
+   $( "#datepicker-start" ).datepicker("show");
+
+   $( "#datepicker-end" ).datepicker();
+   $( "#datepicker-end" ).datepicker("show");
+});
+
+$("#get_parties_button").on('click', function() {
```

Comment:
Can you move these clickhandlers inot `buzzer.js` as well? there's functions for registering other click handlers already there, see `registerUnlinkBuzzerClickHandlers` or `registerBuzzClickHandlers`